.editorconfig

```diff
@@ -230,3 +230,231 @@ dotnet_naming_style.I_style.capitalization = pascal_case
+dotnet_diagnostic.SA1503.severity = warning
+dotnet_diagnostic.SA1504.severity = warning
+dotnet_diagnostic.SA1505.severity = warning
+dotnet_diagnostic.SA1506.severity = warning
+dotnet_diagnostic.SA1507.severity = warning
+dotnet_diagnostic.SA1508.severity = warning
+dotnet_diagnostic.SA1509.severity = warning
+dotnet_diagnostic.SA1510.severity = warning
+dotnet_diagnostic.SA1511.severity = warning
+dotnet_diagnostic.SA1512.severity = warning
+dotnet_diagnostic.SA1513.severity = none
+dotnet_diagnostic.SA1514.severity = warning
+dotnet_diagnostic.SA1515.severity = none
+dotnet_diagnostic.SA1516.severity = none
+dotnet_diagnostic.SA1517.severity = warning
+dotnet_diagnostic.SA1518.severity = warning
+dotnet_diagnostic.SA1519.severity = warning
+dotnet_diagnostic.SA1520.severity = warning
+
+#####################
+# Unused rule areas #
+#####################
+
+# Readability Rules (SA1100-) https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/1.1.118/DOCUMENTATION.md
+
+dotnet_diagnostic.SA1100.severity = none
+dotnet_diagnostic.SA1101.severity = none
+dotnet_diagnostic.SA1102.severity = none
+dotnet_diagnostic.SA1103.severity = none
+dotnet_diagnostic.SA1104.severity = none
+dotnet_diagnostic.SA1105.severity = none
+dotnet_diagnostic.SA1106.severity = none
+dotnet_diagnostic.SA1107.severity = none
+dotnet_diagnostic.SA1108.severity = none
+dotnet_diagnostic.SA1109.severity = none
+dotnet_diagnostic.SA1110.severity = none
+dotnet_diagnostic.SA1111.severity = none
+dotnet_diagnostic.SA1112.severity = none
+dotnet_diagnostic.SA1113.severity = none
+dotnet_diagnostic.SA1114.severity = none
+dotnet_diagnostic.SA1115.severity = none
+dotnet_diagnostic.SA1116.severity = none
+dotnet_diagnostic.SA1117.severity = none
+dotnet_diagnostic.SA1118.severity = none
+dotnet_diagnostic.SA1119.severity = none
+dotnet_diagnostic.SA1120.severity = none
+dotnet_diagnostic.SA1121.severity = none
+dotnet_diagnostic.SA1122.severity = none
+dotnet_diagnostic.SA1123.severity = none
+dotnet_diagnostic.SA1124.severity = none
+dotnet_diagnostic.SA1125.severity = none
+dotnet_diagnostic.SA1126.severity = none
+dotnet_diagnostic.SA1127.severity = none
+dotnet_diagnostic.SA1128.severity = none
+dotnet_diagnostic.SA1129.severity = none
+dotnet_diagnostic.SA1130.severity = none
+dotnet_diagnostic.SA1131.severity = none
+dotnet_diagnostic.SA1132.severity = none
+dotnet_diagnostic.SA1133.severity = none
+dotnet_diagnostic.SA1134.severity = none
+dotnet_diagnostic.SA1135.severity = none
+dotnet_diagnostic.SA1136.severity = none
+dotnet_diagnostic.SA1137.severity = none
+dotnet_diagnostic.SA1138.severity = none
+dotnet_diagnostic.SA1139.severity = none
+dotnet_diagnostic.SA1140.severity = none
+dotnet_diagnostic.SA1141.severity = none
+dotnet_diagnostic.SA1142.severity = none
+
+# Documentation Rules (SA1600-) https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/1.1.118/documentation/DocumentationRules.md
+
+dotnet_diagnostic.SA1600.severity = none
```

Comment:
We don't write too much documentation, but when we do, it would be nice to do it right and to avoid obvious mistakes or useless comment tags. I suggest to activate few safety rules:
SA1603
SA1606
SA1607
SA1612
SA1613
SA1614
SA1616
SA1617
SA1620
SA1621
SA1622
SA1627
SA1628
SA1629